src/gui/mzroll/loginform.cpp

```diff
@@ -57,40 +58,39 @@ void LoginForm::on_pushButton_clicked()
     workerThread->start();
 }
 
-
-void LoginForm::handleResults(QStringList results){
-    QString status_inside=results.at(0);
-    QString username=results.at(1);
-    QString password=results.at(2);
-    if (status_inside=="ok"){
-        qDebug()<<"Logged in, moving on now....";
+void LoginForm::handleResults(QStringList results)
+{
+    QString status_inside = results.at(0);
+    QString username = results.at(1);
```

Comment:
@shubhra-agrawal 
Neither username nor password now needs to be stored. Please remove them.